core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1762,15 +1763,16 @@ write_files:
           - --audit-log-maxbackup=1
           {{ end }}
           {{if .Experimental.Plugins.Rbac.Enabled}}
-          - --authorization-mode=RBAC
-          - --authorization-rbac-super-user=kube-admin
```

Comment:
At first glance I couldn't see why this was removed.
I now suppose it was due to that since k8s 1.6.0 (1) `--authorization-rbac-super-user` has been deprecated and (2) the `system:masters` org provided to our admin certs is enough to make `kube-admin`(the common name of the cert) able to create roles and role bindings. If that's correct, this LGTM 👍